cilium-cli/connectivity/sniff/sniffer.go

```diff
@@ -25,17 +25,111 @@ const (
 	// ModeSanity: expect to observe packets matching the filter, to be
 	// leveraged as a sanity check to verify that the filter is correct.
 	ModeSanity Mode = "sanity"
+
+	// Max wait time for tcpdump to start/stop in remote shell.
+	sniffScriptTimeout = 10 * time.Second
+	// Max timeout before aborting k8s connection in Start/Stop; must exceed sniffScriptTimeout.
+	sniffConnectionTimeout = sniffScriptTimeout + 5*time.Second
+	// Max remote sniffer runtime to prevent lingering processes in the pod.
+	// NOTE: too low may kill tcpdump while test is running.
+	sniffKillTimeout = sniffConnectionTimeout * 4
+
+	// Command executed to start the remote tcpdump in background inside a pod.
+	//
+	// We send tcpdump output to a file inside the pod rather than stdout,
+	// to avoid GH Issue #38643: tcpdump failing with error 14 due to one of
+	// its output streams not being available anymore.
+	// (tcpdump: Unable to write output: Broken pipe)
+	//
+	// Runs silently: succeeds if tcpdump is active ("listening on <iface>"),
+	// else fails with its error code. Includes `timeout 60` to prevent orphaned
```

Comment:
Nit in case of repush:
```suggestion
	// else fails with its error code. Includes `timeout` to prevent orphaned
```
given the timeout is not hardcoded here.